packages/frontend/components/ImageList.tsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react'
+import Image from './Image'
+
+export default function ImageList({ images }) {
```

Comment:
Please add the `type` for `images` variable too, maybe array of type `image` ?

Reply:
fixed